solution_architecture.md

```diff
@@ -269,9 +269,24 @@ If the back end calls from the mobile applications cannot be spread as evenly as
 
 ## Cross-Border Interoperability
 
-A definite prerequisite for compatibility is that the identifiers of the mobile devices can be matched, i.e. the framework by Apple and Google is being used.
+A definite prerequisite for compatibility is that the identifiers of the mobile devices can be matched, i.e. the GAEN framework by Apple and Google is being used.
 
-Further details will be added as soon as they are available.
+Most European countries are developing similar contact tracing apps, these app may use the common frameworks by Google and Apple, enabling transmission and detection of GAEN format diagnosis keys between devices running different contact tracing applications. 
+Each country has its own separate database of users, in order to coordinate exposure information between countries a common service is required.
+The European Federation Gateway Service (EFGS) handles sharing diagnosis keys between different country's backend servers.
+
+![Figure 15: High-level EFGS overview](images/solution_architecture/EFGS_overview.jpg "Figure 15: High-level EFGS overview")
+
+The Federation Gateway Service facilitates backend-to-backend integration, countries can onboard incrementally, while the national backends retain flexibility and control over data distribution to their users.
+For example if a German user is visiting France then all uploaded keys are of relevance to the French database, likewise if a French user is visiting Germany that users keys are of relevance to the German database.
```

Comment:
Suggested rewording:
For example, if a German citizen visits France and then becomes infected, the keys of the German citizen are then relevant for the citizens of France. In this case the German citizen keys would be shared with the EFGS to enable the French backend to obtain the keys. Similarly, if a French user is visiting Germany that users keys are of relevance to the German database.